packages/react/src/components/Listbox/Listbox.tsx

```diff
@@ -65,9 +75,9 @@ const Listbox = forwardRef<HTMLElement, ListboxProps>(
     const [activeOption, setActiveOption] = useState<ListboxOption | null>(
       null
     );
-    const [selectedOption, setSelectedOption] = useState<ListboxOption | null>(
-      null
-    );
+    const [selectedOption, setSelectedOption] = useState<
```

Comment:
Since we potentially want to handle array values, I would change this to be `selectedOptions` and `setSelectedOptions` because the existing naming strategy doesn't make sense for both single/multi select.

Reply:
But in the single select case name `selectedOptions` doesn't make sense either 😅

Reply:
My thinking here was if the internal state is always an array, then it's always "selected options of 1" but I'm open to discussion here.